forge-gui/res/cardsfolder/upcoming/stormkeld_curator_giant_secrets.txt

```diff
@@ -0,0 +1,24 @@
+Name:Stormkeld Curator
+ManaCost:4 W
+Types:Creature Giant
+PT:6/6
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigAurify | TriggerDescription$ When CARDNAME enters the battlefield, you may put any number of Aura cards from your graveyard and/or hand onto the battlefield attached to it.
+SVar:TrigAurify:DB$ RepeatEach | RepeatSubAbility$ BrunaAttach | RepeatCards$ Aura.CanEnchantSource+NotAttachedTo | SubAbility$ HandAuras
+SVar:BrunaAttach:DB$ Attach | Object$ Remembered | Defined$ Self | Optional$ True
```

Comment:
bad copy pasta

Reply:
yeah I fixed that in #3918 – want to just pull this card @Simisays ?